docs/zh/latest/plugins/mocking.md

```diff
@@ -0,0 +1,196 @@
+---
+title: limit-req
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 目录
+
+  - [简介](#简介)
+  - [属性](#属性)
+  - [示例](#示例)
```

Comment:
Please follow the structure of other plugin doc, like https://raw.githubusercontent.com/apache/apisix/master/docs/zh/latest/plugins/gzip.md